example-streamer-uses/src/bin/ue_client.rs

```diff
@@ -73,41 +79,43 @@ async fn main() -> Result<(), UStatus> {
 
     println!("uE_client");
 
-    // TODO: Probably make somewhat configurable?
-    // Create a configuration object
     let mut zenoh_config = Config::default();
 
     if !args.endpoint.is_empty() {
         // Specify the address to listen on using IPv4
-        let ipv4_endpoint = EndPoint::from_str(args.endpoint.as_str());
+        let ipv4_endpoint =
+            EndPoint::from_str(args.endpoint.as_str()).expect("Unable to set endpoint");
 
         // Add the IPv4 endpoint to the Zenoh configuration
         zenoh_config
             .listen
-            .endpoints
-            .push(ipv4_endpoint.expect("FAIL"));
+            .set_endpoints(zenoh::config::ModeDependentValue::Unique(vec![
```

Comment:
That's a mouthful

Reply:
Yeah, maybe should import the enum to shorten the line.